index.html

```diff
@@ -50,38 +50,71 @@ <h1 class="heading-1-dark">Salesforce Certification Checker</h1>
         </section>
 
         <section class="tool">
+            <div class="dropdown">
+                <!-- <select id="roles">
```

Comment:
Is it possible to drop this commented markup?

Reply:
This is now fixed!